### PR TITLE
new selenium project

### PR DESCRIPTION
**Pull request by karatr**

### CODE REVIEW

File: SeleniumProject/OpemText-samples/src/main/java/opentext/mbt/utils/Elements.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package opentext.mbt.utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Elements {
+
+    public static RemoteWebDriver driver;
```
Comment (Zsolt-Lukacs22) [inline SeleniumProject/OpemText-samples/src/main/java/opentext/mbt/utils/Elements.java:10]: I am pretty sure the driver won't be injected here, unless you add an empty test to this class.

File: SeleniumProject/OpemText-samples/pom.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/org.testng/testng -->
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.8.0</version>
+            <scope>test</scope>
```
Comment (Zsolt-Lukacs22) [inline SeleniumProject/OpemText-samples/pom.xml:24]: As a suggestion, you can remove the scope from here, and delete the other testng dependency, so it won't be duplicated.

File: SeleniumProject/OpemText-samples/dependency-reduced-pom.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.example</groupId>
```
Comment (Zsolt-Lukacs22) [inline SeleniumProject/OpemText-samples/dependency-reduced-pom.xml:4]: Can we use our groupId, not this example?

File: SeleniumProject/OpemText-samples/src/main/java/opentext/mbt/utils/Elements.java (hunk 1 above)
Comment (karatr) [reply SeleniumProject/OpemText-samples/src/main/java/opentext/mbt/utils/Elements.java]: added changes

File: SeleniumProject/OpemText-samples/pom.xml (hunk 2 above)
Comment (karatr) [reply SeleniumProject/OpemText-samples/pom.xml]: added changes

File: SeleniumProject/OpemText-samples/dependency-reduced-pom.xml (hunk 3 above)
Comment (karatr) [reply SeleniumProject/OpemText-samples/dependency-reduced-pom.xml]: added changes

**Pull request merged**